lightning/src/sign/mod.rs

```diff
@@ -985,7 +989,7 @@ pub trait ChangeDestinationSource {
 pub struct InMemorySigner {
 	/// Holder secret key in the 2-of-2 multisig script of a channel. This key also backs the
 	/// holder's anchor output in a commitment transaction, if one is present.
-	pub funding_key: SecretKey,
+	funding_key: SecretKey,
```

Comment:
Seems like we only use this internally, so we could mistakenly not tweak it still despite the change in visibility. Though external users would be protected. Would it be worth making some sort of wrapper (to pass to `sign_with_aux_rand`) with a corresponding method to convert to the possibly-tweaked secret key?

Reply:
Thanks, that LGTM.